Cache/trajectories_reference_biomodels/original_copasi_bungay2006a_14_14.tsv

```diff
@@ -0,0 +1,52 @@
+time	II_f	II_l	mIIa_f	mIIa_l	V_f	V_l	Va_f	Va_l	VII_f	VII_l	VIIa_f	VIIa_l	X_f	X_l	Xa_f	Xa_l	APC_f	APC_l	PS_f	PS_l	Vai_f	Vai_l	PC_f	PC_l	TF_l	TF_VIIa_l	TF_VII_l	TF_VIIa_X_l	TF_VIIa_Xa_l	TF_VII_Xa_l	Xa_Va_l	V_Xa_l	IIa_f	V_IIa_l	Xa_Va_II_l	Xa_Va_mIIa_l	APC_PS_l	TFPI_f	AT_f	IIa_AT_f	TFPI_Xa_l	TFPI_Xa_TF_VIIa_l	APC_PS_Va_l	Xa_AT_f	VII_Xa_l	V_mIIa_l	TM_l	IIa_TM_l	IIa_TM_PC_l	mIIa_AT_l	LIPID	alpha2M_l	alpha2M_IIa_l	alpha2M_Xa_l	nnamed: 5
```

Comment:
@paulstapor Question: Are all of these models newly simulated? Or did just their names change? (How?)

Reply:
Nope, I just changed the names to lowercase. That's *so* much easier to track when adapting paths, moving files, and storing information in a tsv...